README.md

```diff
@@ -115,6 +120,23 @@ adding the following exec_properties:
 
 ## Additional resources
 
+- Check out our official documentation for [RBE Setup](https://www.buildbuddy.io/docs/rbe-setup)
 - For more advanced use cases, check out Bazel's [bazel-toolchains repo](https://github.com/bazelbuild/bazel-toolchains) and the [docs on configuring C++ toolchains](https://docs.bazel.build/versions/master/tutorial/cc-toolchain-config.html).
-- Many thanks to the team at Grail who's [LLVM toolchain repo](https://github.com/grailbio/bazel-toolchain) served as the basis for this repo.
+- Many thanks to the team at Grail who's [LLVM toolchain repo](https://github.com/bazel-contrib/toolchains_llvm) served as the basis for this repo.
```

Comment:
It's no longer maintained by or associated with Grail, the original maintainer left them and handed this over to others (including me).

Reply:
Reworded to thanks the maintainers of the repo instead.